client/src/templates/Challenges/classic/editor.tsx

```diff
@@ -651,6 +654,10 @@ const Editor = (props: EditorProps): JSX.Element => {
       };
     }
 
+    // Must manually set test feedback height back to zero since
+    // clearTestFeedback does not.
+    const testStatus = document.getElementById('test-status');
+    if (testStatus) testStatus.style.height = '0';
```

Comment:
Nit: could we create a function for this and replace all the other `testStatus.style.height` resets with it?  It would be nice to have this be the same level of abstraction as `clearTestFeedback`.